<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdycjaTowar</class>
 <widget class="QDialog" name="EdycjaTowar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>207</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>207</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Edycja Towaru</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Nazwa:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Opis:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Ilość:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>Cena:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Vat:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Id Kategorii:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLineEdit" name="lineNazwaT"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineOpisT"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineIloscT"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineCenaT"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineVatT"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineIdKategorT"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Opcje edycji</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonDodajT">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonWyczyscT">
            <property name="text">
             <string>Wyczyść</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonAnulujT">
            <property name="text">
             <string>Anuluj</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
